packages/api/lambdas/sf-event-sqs-to-db-records/write-execution.js

```diff
@@ -130,6 +130,13 @@ const writeExecutionToDynamoAndES = async (params) => {
   }
 };
 
+const publishExecutionSnsMessage = async (record) => {
```

Comment:
This function is used outside of just this lambda, so it should live in a helper file somewhere. Perhaps in `packages/api/lib`?

Reply:
That is a really good point, I will relocate it!